packages/phoenix-event-display/src/loaders/objects/phoenix-objects.ts

```diff
@@ -34,6 +34,9 @@ export class PhoenixObjects {
    */
   public static getTrack(trackParams: any): Object3D {
     let positions = trackParams.pos;
+    if (!positions) {
```

Comment:
The files had tracks with no positions? That's not great.

Reply:
It's this one: https://github.com/HSF/phoenix/blob/master/packages/phoenix-ng/projects/phoenix-app/src/assets/files/event_data/EventData.json

Reply:
![image](https://user-images.githubusercontent.com/36920441/112158452-77972600-8be8-11eb-83e3-bc588ddb2ae1.png)

We already have RungeKutta so may be we should handle this differently.